web_m2x_options/static/src/components/relational_utils.esm.js

```diff
@@ -0,0 +1,224 @@
+/** @odoo-module **/
+
+import {Many2XAutocomplete} from "@web/views/fields/relational_utils";
+import {patch} from "@web/core/utils/patch";
+import {sprintf} from "@web/core/utils/strings";
+import {useService} from "@web/core/utils/hooks";
+const {Component, onWillStart} = owl;
+
+export function is_option_set(option) {
+    if (_.isUndefined(option)) return false;
+    if (typeof option === "string") return option === "true" || option === "True";
+    if (typeof option === "boolean") return option;
+    return false;
+}
+
+patch(Many2XAutocomplete.prototype, "web_m2x_options.Many2XAutocomplete", {
+    setup() {
+        this._super(...arguments);
+        const ormService = useService("orm");
+        this.user_context = Component.env.session.user_context;
+        onWillStart(async () => {
+            this.ir_options = await ormService.call(
+                "ir.config_parameter",
+                "get_web_m2x_options",
+                [],
+                {context: this.user_context}
+            );
+        });
+    },
+
+    async loadOptionsSource(request) {
+        if (this.lastProm) {
+            this.lastProm.abort(false);
+        }
+        // Add options limit used to change number of selections record
+        // returned.
+        if (!_.isUndefined(this.ir_options["web_m2x_options.limit"])) {
+            this.props.searchLimit = parseInt(
+                this.ir_options["web_m2x_options.limit"],
+                10
+            );
+        }
+
+        if (typeof this.props.nodeOptions.limit === "number") {
+            this.props.searchLimit = this.props.nodeOptions.limit;
```

Comment:
The limit option seems to work fine in many2one fields, but not in many2many_tags, this.props.nodeOptions is empty in this widget.
To reproduce in runbot:
- Remove web_m2x_options.limit parameter
- edit sale.order form view adding limit in tag_is field
` <field name="tag_ids" widget="many2many_tags" options="{'color_field': 'color', 'no_create_edit': True, 'limit': 10}"/>`

Reply:
yes, confirmed this issue, I will update soon, thanks

Reply:
Yeah, thanks for the fix, works fine.